tests/Test.hs

```diff
@@ -115,31 +124,10 @@ miniTests =
           , lam $ \f -> lam $ \x -> f `app` x
           )
         , ( "Trivial user macro"
-          , "[let-syntax \n\
-            \  [m [lambda [_] \n\
-            \       [pure [quote [lambda [x] x]]]]] \n\
-            \  m]"
-          , lam $ \x -> x
+          , trivialUserMacro, lam $ \x -> x
```

Comment:
It pains me to see you jump through hoops in order to keep those old tests running, let's get rid of them! #204

Reply:
I think this is better left to another PR